### DISCUSSION TITLE
eBay Partner Network for your eBay store

### DISCUSSION THREAD

**Discussion by NebularNerd** [keirf/greaseweazle#162] [Ideas]
Just a quick suggestion for @keirf have you considered using the [eBay Partner Network](https://partnernetwork.ebay.com/) for linking to your store? It's a handy way to earn a few pennies back from eBay for any sales you make from us going through your links.

There's no extra cost to us as buyers and the little extra back from eBay offsets their fees if nothing else.

**Comment by keirf** [comment]
Good idea thanks. I updated the links on the wiki.

**Comment by NebularNerd** [comment]
You're welcome, I ordered mine a couple of days ago, looking forward to getting my old floppies preserved. 🙂
